index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+
+
+    <title>Jean Michel Basquiat </title>
+    
+    <!--
+    <style type="text/css">
+      body {
+        font-size: 20px;
+      }
+      p.main {
+        color: black;
+      }
+      .main {
+        color: black;
+        font-weight: bold;
+
+      } 
+      -->
+      
+    </style>
```

Comment:
:hammer: Who's this little straggling uncommented-out style tag here? While this probably won't break the html in this case but this could have had other serious side effects if it were a `<div>` or some other tag in the body. Watch out!